apps/broker-service/deploy.ts

```diff
@@ -0,0 +1,11 @@
+import { SetGCloudConfig, copyEnvVarsLocal, copyNpmTokenHere, gCloudDeploy, removeOldAppVersions } from "../../tools/predeploy";
+
+(async () => {
+  await SetGCloudConfig("GCLOUD_BROKER_CONFIG");
+  await copyEnvVarsLocal("app.secrets.yaml");
```

Comment:
is yaml format used elsewhere?

Reply:
yeah, all the gcloud stuff is in yaml.  It's not really 'used', so much as there are just a few files written that way.  I don't think we have any yaml parsers or anything.